content/docs/services/cloud-gov-service-account.md

```diff
@@ -44,11 +44,39 @@ cf create-service-key my-service-account my-service-key
 cf service-key my-service-account my-service-key
 ```
 
+The last command will return a username/password pair, that you can use, like this:
+
+```
+{
+ "password": "oYasdfliaweinasfdliecV",
+ "username": "deadbeed-aabb-1234-feha0987654321000"
+}
+```
+
 This will create a cloud.gov service account and make the credentials available to you via a service key. Keep these credentials secure. If they’re compromised, the way to invalidate the credentials is to delete the service key (you can create another, and it will have a fresh set of credentials). Each service key that you bind to your instance creates a separate service account with different credentials; you can create as many service keys per instance as you like. <!-- this advice should match on /docs/services/cloud-gov-service-account/ + /docs/services/cloud-gov-identity-provider/ -->
```

Comment:
Suggest linking https://docs.cloudfoundry.org/devguide/services/service-keys.html#delete whenever we mention "delete the service key", since people may read this in a panic after a compromise and want a quick link to the instructions (without reading further on the page).